_source/api/logzio-public-api.yml

```diff
@@ -5601,6 +5808,62 @@ definitions:
         description: Detailed description of the endpoint
         example: Endpoint for sending alerts to Slack
 
+  OpsGenieEndpointUpsertRequest:
+    type: object
+    properties:
+      title:
+        type: string
+        description: Name of the endpoint
+        example: OpsGenie endpoint
+      description:
+        type: string
+        description: Detailed description of the endpoint
+        example: Sends notifications to OpsGenie
+      apiKey:
+        type: string
+        description: API key from OpsGenie
+        example: c687f9231619d7d7b959f33e4cc821a5
+
+  ServiceNowEndpointUpsertRequest:
+    type: object
+    properties:
+      title:
+        type: string
+        description: Name of the endpoint
+        example: New ServiceNow endpoint
+      description:
+        type: string
+        description: Detailed description of the endpoint
+        example: Sends notifications to logzio-alerts channel
+      username:
+        type: string
+        description: ServiceNow user name
+        example: User
+      password:
+        type: string
+        description: ServiceNow password
+        example: Password
+      url:
+        type: string
+        description: Your ServiceNow webhook URL
```

Comment:
Provide your instance url to connect to your existing ServiceNow instance. i.e. https://xxxxxxxxx.service-now.com/